src/i18n/locale/en.json

```diff
@@ -360,7 +360,7 @@
       "label": {
         "build_time": "Build time",
         "discord": "Discord",
-        "github": "Github",
+        "github": "GitHub",
         "server": "Server",
         "server_address": "Server Address",
```

Comment:
```suggestion
        "server_address": "Server address",
```

Reply:
proper noun? I'm not sure about this one